repose-aggregator/src/docs/asciidoc/filters/cors.adoc

```diff
@@ -1,20 +1,27 @@
-= CORS
-:toc:
-:icons: font
+= Cross-Origin Resource Sharing (CORS) Filter
 
 include::../_includes/in-progress.adoc[]
 
 The Cross-Origin Resource Sharing (CORS) filter allows Repose to manage CORS requests without the origin service needing to understand CORS.
-For an introduction to CORS, see the <<CORS Overview>> below.
+For an introduction to CORS, see the <<CORS Overview>> section below.
 
-WARNING: Before enabling the CORS filter, you should be familiar with the CORS specification and the associated security implications.
+[WARNING]
+====
+Before enabling the CORS filter, you should be familiar with the https://www.w3.org/TR/cors/[CORS specification] and the associated security implications.
 Enabling CORS can increase an endpoint's exposure.
-Configuring CORS improperly could expose CSRF vulnerabilities in the origin service that were previously blocked by not supporting CORS.
-
-== Headers
-
-=== Required
-The presence of certain headers will indicate what kind of request is being made.
+Configuring CORS improperly could expose https://en.wikipedia.org/wiki/Cross-site_request_forgery[Cross-Site Request Forgery (CSRF)] vulnerabilities in the origin service that were previously hidden by not supporting CORS.
+====
+
+== General filter information
```

Comment:
Hm, that's a question. On some pages we capitalize all words other than articles while on others we only capitalize the first word. We should probably try to be consistent on section titles, so which do we prefer? That should also be in the style guide.